src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = FigureSupplier.getRandomFigure();
```

Comment:
Don't use static methods in your solution. Instead of calling `FigureSupplier.getRandomFigure()` statically, you should call the method on the instance of `FigureSupplier` that you created on line 6.